docs/notebooks/Getting_started_1.ipynb

```diff
@@ -0,0 +1,440 @@
+    "3. [🔨 Design rules for 1-Lipschitz neural networks with `deel-lip`](#design)   \n",
+    "A set of neural network design rules that one must respect in order to enforce the 1-Lipschitz constraint.\n",
+    "\n",
+    "\n",
+    "\n",
+    "## 📚 Theoretical background <a id='theoretical_background'></a> <a name='theoretical_background'></a>\n",
+    "### What is a Lipschitz constant\n",
+    "The `deel-lip` package allows to control the Lipschitz constant of a layer or of a whole neural network. The Lipschitz constant is a mathematical property of a function (in our context of work, a layer or a model) that characterizes how much the output of the function can change with respect to changes in its input. \n",
+    "\n",
+    "In mathematical terms, a function $f$ is Lipschitz continuous with a **Lipschitz constant L** or more simply **L-Lipschitz** if for any given pair of points $x_1,x_2$, $L$ provides a bound on the rate of change of $f$:  \n",
+    "\n",
+    "$$||f(x_1)-f(x_2)||\\leq L||x_1-x_2||.$$\n",
+    "\n",
+    "For instance, given a 1-Lipschitz dense layer (a.k.a fully connected layer) with a weight matrix $W$ and a bias vector $b$, we have for any two inputs $x_1$ and $x_2$: $$||W.x_1+b-(W.x_2+b)|| \\leq 1||x_1-x_2||.$$\n",
+    "\n",
+    "💡 The norm we refer to throughout our notebooks is the Euclidean norm (L2). This is because `deel-lip` operates with this norm. You will find more information about the role of the norm in the context of adversarially robust 1-Lipschitz deep learning models in the notebook titled 'Getting Started 2'.\n",
+    "\n",
+    "### A simple requirement for creating 1-Lipschitz neural network\n",
+    "The composition property of Lipschitz continuous functions states that if you have a function f that is $L_1$-Lipschitz and another function g that is $L_2$-Lispchitz, then their composition function h = (f o g) which applies f after g is also Lipschitz continuous with a Lipschitz constant $L \\leq L_1$ * $L_2$.\n",
+    "\n",
+    "A feed-forward or sequential neural network is essentially a stack of layers, where each layer transforms the output of the previous layer(s) and feeds its output to the next ones. \n",
+    "\n",
+    "By the composition property of Lipschitz functions, *it suffices for each of the n individual layers of a neural network model to be 1-Lipschitz, for the whole model to be 1-Lipschitz*.\n",
+    "\n",
+    "For instance, given a 1-Lipschitz dense layer parametrized by $(W_1,b_1)$, and a ReLU (Rectified Linear Unit) activation layer which is naturally 1-Lipschitz, the combination of the two is also 1-Lispchitz.   \n",
+    "This is shown in the equations below, where we have for any two inputs $x_1$ and $x_2$:\n",
+    "\n",
+    "$$||W_1.x_1+b_1-(W_1.x_2+b_1)||\\leq 1||x_1-x_2||,$$\n",
+    "$$||ReLU(x_1)-ReLU(x_2)||\\leq 1||x_1-x_2||,$$\n",
+    "and:\n",
+    "$$||ReLU(W_1.x_1+b_1)-ReLU(W_1.x_2+b_1)||\\leq 1||W_1.x_1+b_1-(W_1.x_2+b_1)||\\leq 1^2||x_1-x_2||.$$\n",
```

Comment:
To be consistent with the previous equation and to have a simpler notation, it might be better to keep $(W, b)$ instead of $(W_1, b_1)$.
Moreover, adding parentheses around $(W.x_1+b)$ in  $$||(W.x_1+b)-(W.x_2+b)|| could emphasize the $f(x_1)$ vs $f(x_2)$.
The suggestions below:
```suggestion
    "For instance, given a 1-Lipschitz dense layer parametrized by $(W,b)$, and a ReLU (Rectified Linear Unit) activation layer which is naturally 1-Lipschitz, the combination of the two is also 1-Lispchitz.   \n",
    "This is shown in the equations below, where we have for any two inputs $x_1$ and $x_2$:\n",
    "\n",
    "$$||(W.x_1+b)-(W.x_2+b)||\\leq 1||x_1-x_2||,$$\n",
    "$$||ReLU(x_1)-ReLU(x_2)||\\leq 1||x_1-x_2||,$$\n",
    "and:\n",
    "$$||ReLU(W.x_1+b)-ReLU(W.x_2+b)||\\leq 1||(W.x_1+b)-(W.x_2+b)||\\leq 1^2||x_1-x_2||.$$\n",
```